### PR TITLE
Centered Grid

### PR DESCRIPTION
**Pull request by ShubhamJoshi-1994**
Implemented Centered Grid Element with Heading.

### CODE REVIEW

File: packages/react/src/organisms/cards/IconTitleDescriptionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      </div>
+    );
+  } else {
+    return null;
+  }
+};
+
+IconTitleDescriptionCard.defaultProps = {
+  logoPosition: "left", // top-left, left
+  logoWrapperClasses:
```
Comment (rjvim) [inline packages/react/src/organisms/cards/IconTitleDescriptionCard.tsx:76]: @ShubhamJoshi-1994 logoWrapperClasses should come from theme

File: packages/react/src/organisms/sectionHeader/base.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  actions?: any;
+  horizontalPadding?: string;
+  verticalPadding?: string;
+  backgroundClasses?: string;
+  widthClasses?: string;
+}
+
+const SectionHeaderBase = (props: SectionHeaderBaseProps) => {
+  const backgroundClasses = props.backgroundClasses || "bg-gray-50";
+  const widthClasses =
```
Comment (rjvim) [inline packages/react/src/organisms/sectionHeader/base.tsx:20]: @ShubhamJoshi-1994 widthClasses and backgroundClasses should come from theme.

Just think this way, a developer might pass overriding props everytime they use SectionHeaderBase, or they might just create a variant one place and pass "variant" prop. Variant prop is more flexible, that's why styles should be resolved using resolveProps and shouldn't be passed like above.

File: packages/react/src/organisms/cards/IconTitleDescriptionCard.tsx (hunk 1 above)
Comment (ShubhamJoshi-1994) [reply packages/react/src/organisms/cards/IconTitleDescriptionCard.tsx]: Resolved. Also removed `defaultProps` using theme classes now.

File: packages/react/src/organisms/sectionHeader/base.tsx (hunk 2 above)
Comment (ShubhamJoshi-1994) [reply packages/react/src/organisms/sectionHeader/base.tsx]: This file is removed, removed hard coded classes from other components, using theme file.

**Pull request merged**